### ISSUE TITLE
Area label is truncated when there are not enough items for it

### ISSUE THREAD

**Issue by Platonn** [PGS-dev/technology-radar-ui#8]
![issue](https://user-images.githubusercontent.com/4001059/27934270-43186a8a-62a5-11e7-9818-74279a94bfd6.png)
![issue2](https://user-images.githubusercontent.com/4001059/27934332-8160b400-62a5-11e7-9195-08ef12bebf3f.png)

**Comment by Platonn** [comment]
In `chart.js` in lines `146-147` there is caluculation of angleStart and angleEnd. Those values shouldn't be too close. Based on them there are calculated "widths" of labels arcs and texts inside of them.

:idea: @amortka What do you think about duplicating Area Labels, but with enlarged angles (start and end) and displaying them only on mouse over event? Or maybe you can see a simpler solution?

**Comment by amortka** [comment]
It has been fixed with commit #634be3783a86813f6d42de375846957f1d0360df

**Comment by amortka** [comment]
with ?debug=true&limit=<n> it's possible to simulate low amount of blips:

![image](https://user-images.githubusercontent.com/8505274/28418101-d38782c6-6d5a-11e7-9d6e-9f63eb86bb2b.png)

![image](https://user-images.githubusercontent.com/8505274/28418122-eadba682-6d5a-11e7-9599-a8a9093e456f.png)

Fix:
![image](https://user-images.githubusercontent.com/8505274/28418137-f7101370-6d5a-11e7-87c3-3dd775f1882a.png)
![image](https://user-images.githubusercontent.com/8505274/28418141-00160bf0-6d5b-11e7-8ef3-c5b23f0b7e7d.png)

Only those labels that fit should be displayed, alse see function textPercentWidth to generate text proper length:

```javascript
      .text((d, idx) => {
        const textWidth = getTextLength(d.name, this.svg);
        const pathWidth = document.getElementById('Section--labelArc_' + idx).getTotalLength();
        const percent = Math.floor(pathWidth / textWidth * 0.8 * 100) / 100;

        return textWidth > pathWidth ? 
         pathWidth > 10 ? `${textPercentWidth(d.name, percent)}...` : '' :
         d.name;
      });
```
